src/components/cylc/tree/TreeItem.vue

```diff
@@ -170,7 +170,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
                 class="leaf-entry output px-5"
               >
                 <span class="leaf-entry-title">{{ customOutput.label }}</span>
-                <span class="text-grey leaf-entry-value">{{ customOutput.message }}</span>
+                <span class="text-grey-darken-1 leaf-entry-value">{{ customOutput.message }}</span>
```

Comment:
could we have dark blue so as to have a little more contrast?

Reply:
Actually I've noticed this section can get a bit messy with long message strings. We could just use a Vuetify table component to display these, opening a new issue

Reply:
Anything working on Cylc message is going to be at some risk of such messiness. I believe it's well protected against injection attacks.... but otherwise...

Reply:
New issue: #1458